[2024-10-11T13:45:01.623+0000] {processor.py:186} INFO - Started process (PID=70) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:45:01.673+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:45:01.754+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:45:01.750+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:45:03.041+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:45:02.862+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:45:03.056+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:45:03.264+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 1.766 seconds
[2024-10-11T13:45:33.535+0000] {processor.py:186} INFO - Started process (PID=139) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:45:33.536+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:45:33.539+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:45:33.539+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:45:33.568+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:45:33.561+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:45:33.570+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:45:33.606+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.078 seconds
[2024-10-11T13:46:03.944+0000] {processor.py:186} INFO - Started process (PID=202) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:46:03.946+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:46:03.951+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:46:03.951+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:46:03.979+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:46:03.973+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:46:03.981+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:46:04.010+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.073 seconds
[2024-10-11T13:47:07.895+0000] {processor.py:186} INFO - Started process (PID=65) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:47:07.896+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:47:07.899+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:47:07.899+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:47:07.921+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:47:07.916+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:47:07.923+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:47:07.943+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.053 seconds
[2024-10-11T13:47:38.591+0000] {processor.py:186} INFO - Started process (PID=128) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:47:38.592+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:47:38.595+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:47:38.595+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:47:38.707+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:47:38.622+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:47:38.709+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:47:38.741+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.156 seconds
[2024-10-11T13:48:09.396+0000] {processor.py:186} INFO - Started process (PID=191) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:48:09.398+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:48:09.411+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:48:09.410+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:48:09.492+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:48:09.484+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:48:09.496+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:48:09.543+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.155 seconds
[2024-10-11T13:48:39.830+0000] {processor.py:186} INFO - Started process (PID=260) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:48:39.831+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:48:39.834+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:48:39.833+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:48:39.867+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:48:39.862+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:48:39.869+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:48:39.895+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.072 seconds
[2024-10-11T13:49:10.766+0000] {processor.py:186} INFO - Started process (PID=323) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:49:10.767+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:49:10.770+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:49:10.770+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:49:10.805+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:49:10.793+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:49:10.808+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:49:10.849+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.108 seconds
[2024-10-11T13:49:41.748+0000] {processor.py:186} INFO - Started process (PID=386) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:49:41.749+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:49:41.751+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:49:41.751+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:49:41.782+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:49:41.776+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:49:41.784+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:49:41.813+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.070 seconds
[2024-10-11T13:50:12.647+0000] {processor.py:186} INFO - Started process (PID=449) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:50:12.648+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:50:12.650+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:50:12.650+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:50:12.682+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:50:12.677+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:50:12.684+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:50:12.705+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.063 seconds
[2024-10-11T13:50:43.113+0000] {processor.py:186} INFO - Started process (PID=512) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:50:43.115+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:50:43.121+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:50:43.120+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:50:43.170+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:50:43.163+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:50:43.173+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:50:43.193+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.086 seconds
[2024-10-11T13:51:13.606+0000] {processor.py:186} INFO - Started process (PID=575) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:51:13.607+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:51:13.611+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:51:13.610+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:51:13.648+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:51:13.643+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:51:13.650+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:51:13.673+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.073 seconds
[2024-10-11T13:51:44.668+0000] {processor.py:186} INFO - Started process (PID=638) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:51:44.669+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:51:44.673+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:51:44.673+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:51:44.703+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:51:44.697+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:51:44.705+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:51:44.737+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.074 seconds
[2024-10-11T13:52:15.741+0000] {processor.py:186} INFO - Started process (PID=701) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:52:15.742+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:52:15.746+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:52:15.745+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:52:15.792+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:52:15.784+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:52:15.796+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:52:15.826+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.092 seconds
[2024-10-11T13:52:46.031+0000] {processor.py:186} INFO - Started process (PID=764) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:52:46.032+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:52:46.034+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:52:46.034+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:52:46.063+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:52:46.058+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:52:46.065+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:52:46.081+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.055 seconds
[2024-10-11T13:53:17.072+0000] {processor.py:186} INFO - Started process (PID=827) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:53:17.073+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:53:17.075+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:53:17.075+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:53:17.104+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:53:17.099+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from dags.scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'dags'
[2024-10-11T13:53:17.107+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:53:17.128+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.061 seconds
[2024-10-11T13:53:35.784+0000] {processor.py:186} INFO - Started process (PID=884) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:53:35.785+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:53:35.787+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:53:35.787+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:53:35.844+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:53:35.837+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:53:35.846+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:53:35.866+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.088 seconds
[2024-10-11T13:54:06.634+0000] {processor.py:186} INFO - Started process (PID=947) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:54:06.636+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:54:06.640+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:54:06.639+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:54:06.680+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:54:06.672+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:54:06.682+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:54:06.710+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.081 seconds
[2024-10-11T13:54:37.290+0000] {processor.py:186} INFO - Started process (PID=1010) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:54:37.292+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:54:37.294+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:54:37.294+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:54:37.331+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:54:37.324+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:54:37.334+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:54:37.352+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.066 seconds
[2024-10-11T13:55:07.463+0000] {processor.py:186} INFO - Started process (PID=1073) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:55:07.464+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:55:07.468+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:55:07.468+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:55:07.502+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:55:07.496+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:55:07.503+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:55:07.518+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.060 seconds
[2024-10-11T13:55:37.856+0000] {processor.py:186} INFO - Started process (PID=1136) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:55:37.858+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:55:37.861+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:55:37.860+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:55:37.894+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:55:37.889+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:55:37.896+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:55:38.783+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.934 seconds
[2024-10-11T13:56:09.202+0000] {processor.py:186} INFO - Started process (PID=1199) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:56:09.205+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:56:09.214+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:56:09.213+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:56:09.321+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:56:09.312+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:56:09.325+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:56:09.354+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.161 seconds
[2024-10-11T13:56:39.897+0000] {processor.py:186} INFO - Started process (PID=1262) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:56:39.899+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:56:39.904+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:56:39.903+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:56:39.932+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:56:39.928+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 10, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:56:39.934+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:56:39.951+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.061 seconds
[2024-10-11T13:56:49.059+0000] {processor.py:186} INFO - Started process (PID=1269) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:56:49.061+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:56:49.064+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:56:49.064+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:56:49.113+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:56:49.107+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:56:49.116+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:56:49.136+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.081 seconds
[2024-10-11T13:57:42.420+0000] {processor.py:186} INFO - Started process (PID=59) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:57:42.424+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:57:42.437+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:57:42.435+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:57:42.477+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:57:42.465+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:57:42.483+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:57:42.576+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.177 seconds
[2024-10-11T13:58:13.479+0000] {processor.py:186} INFO - Started process (PID=128) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:58:13.481+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:58:13.482+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:58:13.482+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:58:13.507+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:58:13.502+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:58:13.510+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:58:13.529+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.055 seconds
[2024-10-11T13:58:43.781+0000] {processor.py:186} INFO - Started process (PID=191) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:58:43.782+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:58:43.785+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:58:43.784+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:58:43.805+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:58:43.801+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:58:43.807+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:58:43.824+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.047 seconds
[2024-10-11T13:59:14.323+0000] {processor.py:186} INFO - Started process (PID=254) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:59:14.324+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:59:14.329+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:59:14.328+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:59:14.359+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:59:14.354+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:59:14.361+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:59:14.401+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.089 seconds
[2024-10-11T13:59:44.713+0000] {processor.py:186} INFO - Started process (PID=317) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:59:44.714+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T13:59:44.717+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:59:44.716+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:59:44.743+0000] {logging_mixin.py:190} INFO - [2024-10-11T13:59:44.737+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T13:59:44.745+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T13:59:44.780+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.071 seconds
[2024-10-11T14:00:14.860+0000] {processor.py:186} INFO - Started process (PID=380) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:00:14.862+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:00:14.865+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:00:14.864+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:00:14.896+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:00:14.891+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:00:14.898+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:00:14.922+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.067 seconds
[2024-10-11T14:00:45.453+0000] {processor.py:186} INFO - Started process (PID=443) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:00:45.454+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:00:45.457+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:00:45.456+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:00:45.488+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:00:45.481+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:00:45.491+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:00:45.509+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.061 seconds
[2024-10-11T14:01:15.838+0000] {processor.py:186} INFO - Started process (PID=506) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:01:15.840+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:01:15.844+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:01:15.843+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:01:15.871+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:01:15.866+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:01:15.874+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:01:15.892+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.061 seconds
[2024-10-11T14:01:46.838+0000] {processor.py:186} INFO - Started process (PID=569) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:01:46.840+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:01:46.842+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:01:46.842+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:01:46.870+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:01:46.864+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:01:46.872+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:01:46.891+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.057 seconds
[2024-10-11T14:02:17.485+0000] {processor.py:186} INFO - Started process (PID=632) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:02:17.488+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:02:17.494+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:02:17.492+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:02:17.522+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:02:17.515+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:02:17.524+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:02:17.549+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.073 seconds
[2024-10-11T14:02:48.478+0000] {processor.py:186} INFO - Started process (PID=695) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:02:48.480+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:02:48.484+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:02:48.484+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:02:48.513+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:02:48.506+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:02:48.515+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:02:48.539+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.069 seconds
[2024-10-11T14:03:18.816+0000] {processor.py:186} INFO - Started process (PID=758) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:03:18.818+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:03:18.822+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:03:18.821+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:03:18.862+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:03:18.853+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:03:18.865+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:03:18.896+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.087 seconds
[2024-10-11T14:03:49.662+0000] {processor.py:186} INFO - Started process (PID=821) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:03:49.663+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:03:49.667+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:03:49.667+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:03:49.697+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:03:49.689+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:03:49.701+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:03:49.720+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.067 seconds
[2024-10-11T14:04:20.200+0000] {processor.py:186} INFO - Started process (PID=884) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:04:20.202+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:04:20.208+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:04:20.208+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:04:20.247+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:04:20.238+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:04:20.251+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:04:20.280+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.090 seconds
[2024-10-11T14:04:50.649+0000] {processor.py:186} INFO - Started process (PID=947) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:04:50.651+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:04:50.653+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:04:50.653+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:04:50.679+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:04:50.674+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:04:50.682+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:04:50.706+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.064 seconds
[2024-10-11T14:05:21.039+0000] {processor.py:186} INFO - Started process (PID=1010) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:05:21.040+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:05:21.042+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:05:21.042+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:05:21.070+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:05:21.064+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:05:21.072+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:05:21.095+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.063 seconds
[2024-10-11T14:05:51.948+0000] {processor.py:186} INFO - Started process (PID=1073) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:05:51.950+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:05:51.952+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:05:51.952+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:05:51.985+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:05:51.979+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:05:51.988+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:05:52.018+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.075 seconds
[2024-10-11T14:06:22.209+0000] {processor.py:186} INFO - Started process (PID=1136) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:06:22.213+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:06:22.218+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:06:22.217+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:06:22.246+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:06:22.240+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:06:22.249+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:06:22.285+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.087 seconds
[2024-10-11T14:06:53.211+0000] {processor.py:186} INFO - Started process (PID=1199) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:06:53.214+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:06:53.220+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:06:53.219+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:06:53.249+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:06:53.242+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:06:53.251+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:06:53.271+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.068 seconds
[2024-10-11T14:07:24.181+0000] {processor.py:186} INFO - Started process (PID=1263) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:07:24.182+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:07:24.188+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:07:24.187+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:07:24.217+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:07:24.210+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:07:24.219+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:07:24.242+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.079 seconds
[2024-10-11T14:07:54.741+0000] {processor.py:186} INFO - Started process (PID=1326) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:07:54.743+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:07:54.746+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:07:54.746+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:07:54.769+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:07:54.764+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:07:54.771+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:07:54.793+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.057 seconds
[2024-10-11T14:08:25.512+0000] {processor.py:186} INFO - Started process (PID=1389) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:08:25.514+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:08:25.518+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:08:25.518+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:08:25.550+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:08:25.545+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:08:25.552+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:08:25.578+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.074 seconds
[2024-10-11T14:08:56.209+0000] {processor.py:186} INFO - Started process (PID=1452) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:08:56.211+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:08:56.215+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:08:56.214+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:08:56.243+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:08:56.237+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:08:56.245+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:08:56.283+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.080 seconds
[2024-10-11T14:09:26.396+0000] {processor.py:186} INFO - Started process (PID=1515) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:09:26.397+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:09:26.399+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:09:26.399+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:09:26.422+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:09:26.417+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:09:26.424+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:09:26.446+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.055 seconds
[2024-10-11T14:09:57.119+0000] {processor.py:186} INFO - Started process (PID=1578) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:09:57.121+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:09:57.123+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:09:57.123+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:09:57.147+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:09:57.142+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:09:57.150+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:09:57.179+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.065 seconds
[2024-10-11T14:10:27.827+0000] {processor.py:186} INFO - Started process (PID=1641) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:10:27.828+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:10:27.830+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:10:27.830+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:10:27.858+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:10:27.853+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:10:27.860+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:10:27.882+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.060 seconds
[2024-10-11T14:10:58.856+0000] {processor.py:186} INFO - Started process (PID=1704) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:10:58.859+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:10:58.868+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:10:58.867+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:10:58.904+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:10:58.893+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:10:58.908+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:10:59.053+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.220 seconds
[2024-10-11T14:11:29.607+0000] {processor.py:186} INFO - Started process (PID=1768) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:11:29.608+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:11:29.612+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:11:29.611+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:11:29.640+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:11:29.634+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:11:29.642+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:11:29.672+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.071 seconds
[2024-10-11T14:11:59.941+0000] {processor.py:186} INFO - Started process (PID=1831) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:11:59.942+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:11:59.945+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:11:59.945+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:11:59.976+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:11:59.970+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:11:59.978+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:12:00.017+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.080 seconds
[2024-10-11T14:12:30.278+0000] {processor.py:186} INFO - Started process (PID=1894) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:12:30.279+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:12:30.281+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:12:30.281+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:12:30.302+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:12:30.298+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:12:30.304+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:12:30.324+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.052 seconds
[2024-10-11T14:13:00.753+0000] {processor.py:186} INFO - Started process (PID=1957) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:13:00.755+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:13:00.759+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:13:00.758+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:13:00.781+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:13:00.776+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:13:00.783+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:13:00.806+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.057 seconds
[2024-10-11T14:13:31.772+0000] {processor.py:186} INFO - Started process (PID=2021) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:13:31.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:13:31.776+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:13:31.776+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:13:31.801+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:13:31.795+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:13:31.803+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:13:31.833+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.067 seconds
[2024-10-11T14:14:02.380+0000] {processor.py:186} INFO - Started process (PID=2084) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:14:02.381+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:14:02.384+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:14:02.383+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:14:02.411+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:14:02.404+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:14:02.413+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:14:02.448+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.074 seconds
[2024-10-11T14:14:33.161+0000] {processor.py:186} INFO - Started process (PID=2147) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:14:33.162+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:14:33.165+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:14:33.165+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:14:33.194+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:14:33.188+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:14:33.196+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:14:33.214+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.061 seconds
[2024-10-11T14:15:03.582+0000] {processor.py:186} INFO - Started process (PID=2210) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:15:03.583+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:15:03.586+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:15:03.586+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:15:03.611+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:15:03.607+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:15:03.613+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:15:03.632+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.054 seconds
[2024-10-11T14:15:34.056+0000] {processor.py:186} INFO - Started process (PID=2273) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:15:34.060+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:15:34.068+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:15:34.068+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:15:34.094+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:15:34.088+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:15:34.096+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:15:34.117+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.066 seconds
[2024-10-11T14:16:38.283+0000] {processor.py:186} INFO - Started process (PID=64) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:16:38.295+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:16:38.327+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:16:38.325+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:16:38.413+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:16:38.384+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:16:38.418+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:16:38.534+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.327 seconds
[2024-10-11T14:17:08.775+0000] {processor.py:186} INFO - Started process (PID=134) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:17:08.777+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:17:08.789+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:17:08.788+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:17:08.864+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:17:08.857+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:17:08.866+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:17:08.907+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.147 seconds
[2024-10-11T14:17:39.286+0000] {processor.py:186} INFO - Started process (PID=197) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:17:39.287+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:17:39.290+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:17:39.290+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:17:39.330+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:17:39.322+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:17:39.332+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:17:39.351+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.071 seconds
[2024-10-11T14:18:09.857+0000] {processor.py:186} INFO - Started process (PID=260) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:18:09.859+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:18:09.862+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:18:09.861+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:18:09.888+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:18:09.884+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:18:09.890+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:18:09.905+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.057 seconds
[2024-10-11T14:18:40.278+0000] {processor.py:186} INFO - Started process (PID=323) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:18:40.280+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:18:40.282+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:18:40.282+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:18:40.310+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:18:40.304+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:18:40.312+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:18:40.325+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.051 seconds
[2024-10-11T14:19:10.983+0000] {processor.py:186} INFO - Started process (PID=386) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:19:10.985+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:19:10.988+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:19:10.988+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:19:11.015+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:19:11.009+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:19:11.018+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:19:11.044+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.067 seconds
[2024-10-11T14:19:41.732+0000] {processor.py:186} INFO - Started process (PID=449) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:19:41.733+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:19:41.738+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:19:41.737+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:19:41.763+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:19:41.759+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:19:41.765+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:19:41.790+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.064 seconds
[2024-10-11T14:20:12.720+0000] {processor.py:186} INFO - Started process (PID=512) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:20:12.721+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:20:12.724+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:20:12.724+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:20:12.743+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:20:12.740+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:20:12.745+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:20:12.760+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.044 seconds
[2024-10-11T14:20:42.850+0000] {processor.py:186} INFO - Started process (PID=575) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:20:42.851+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:20:42.853+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:20:42.853+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:20:42.875+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:20:42.870+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:20:42.877+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:20:42.899+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.053 seconds
[2024-10-11T14:21:13.370+0000] {processor.py:186} INFO - Started process (PID=638) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:21:13.371+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:21:13.374+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:21:13.374+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:21:13.400+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:21:13.396+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:21:13.402+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:21:13.422+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.057 seconds
[2024-10-11T14:21:43.807+0000] {processor.py:186} INFO - Started process (PID=700) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:21:43.808+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:21:43.811+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:21:43.811+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:21:43.840+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:21:43.836+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:21:43.842+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:21:43.870+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.072 seconds
[2024-10-11T14:22:14.422+0000] {processor.py:186} INFO - Started process (PID=763) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:22:14.424+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:22:14.428+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:22:14.427+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:22:14.474+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:22:14.468+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:22:14.476+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:22:14.499+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.083 seconds
[2024-10-11T14:22:44.562+0000] {processor.py:186} INFO - Started process (PID=826) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:22:44.563+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:22:44.566+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:22:44.565+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:22:44.590+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:22:44.586+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:22:44.592+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:22:44.612+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.056 seconds
[2024-10-11T14:23:15.624+0000] {processor.py:186} INFO - Started process (PID=889) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:23:15.626+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:23:15.629+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:23:15.628+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:23:15.663+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:23:15.656+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:23:15.666+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:23:15.696+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.079 seconds
[2024-10-11T14:23:45.796+0000] {processor.py:186} INFO - Started process (PID=952) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:23:45.797+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:23:45.806+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:23:45.805+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:23:45.837+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:23:45.832+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:23:45.839+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:23:45.859+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.071 seconds
[2024-10-11T14:24:16.527+0000] {processor.py:186} INFO - Started process (PID=1015) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:24:16.528+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:24:16.532+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:24:16.532+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:24:16.559+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:24:16.555+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:24:16.561+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:24:16.584+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.063 seconds
[2024-10-11T14:24:46.732+0000] {processor.py:186} INFO - Started process (PID=1078) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:24:46.734+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:24:46.736+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:24:46.735+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:24:46.762+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:24:46.755+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:24:46.764+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:24:46.786+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.059 seconds
[2024-10-11T14:25:17.466+0000] {processor.py:186} INFO - Started process (PID=1141) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:25:17.467+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:25:17.469+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:25:17.469+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:25:17.493+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:25:17.487+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:25:17.495+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:25:17.513+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.052 seconds
[2024-10-11T14:25:48.009+0000] {processor.py:186} INFO - Started process (PID=1204) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:25:48.010+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:25:48.012+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:25:48.012+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:25:48.042+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:25:48.036+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:25:48.045+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:25:48.064+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.059 seconds
[2024-10-11T14:26:18.301+0000] {processor.py:186} INFO - Started process (PID=1267) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:26:18.302+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:26:18.305+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:26:18.304+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:26:18.352+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:26:18.331+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:26:18.356+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:26:18.375+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.079 seconds
[2024-10-11T14:26:49.031+0000] {processor.py:186} INFO - Started process (PID=1331) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:26:49.032+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:26:49.034+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:26:49.033+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:26:49.055+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:26:49.050+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:26:49.057+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:26:49.086+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.059 seconds
[2024-10-11T14:27:19.856+0000] {processor.py:186} INFO - Started process (PID=1394) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:27:19.858+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:27:19.865+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:27:19.864+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:27:19.891+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:27:19.885+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:27:19.893+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:27:19.923+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.077 seconds
[2024-10-11T14:27:50.607+0000] {processor.py:186} INFO - Started process (PID=1457) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:27:50.609+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:27:50.612+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:27:50.612+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:27:50.637+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:27:50.632+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:27:50.640+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:27:50.668+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.068 seconds
[2024-10-11T14:28:21.221+0000] {processor.py:186} INFO - Started process (PID=1520) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:28:21.223+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:28:21.226+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:28:21.226+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:28:21.257+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:28:21.251+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:28:21.260+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:28:21.281+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.066 seconds
[2024-10-11T14:28:51.353+0000] {processor.py:186} INFO - Started process (PID=1583) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:28:51.354+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:28:51.357+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:28:51.357+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:28:51.376+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:28:51.372+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:28:51.378+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:28:51.396+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.047 seconds
[2024-10-11T14:29:21.609+0000] {processor.py:186} INFO - Started process (PID=1646) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:29:21.610+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:29:21.615+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:29:21.614+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:29:21.639+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:29:21.633+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:29:21.641+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:29:21.660+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.056 seconds
[2024-10-11T14:29:51.819+0000] {processor.py:186} INFO - Started process (PID=1709) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:29:51.820+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:29:51.823+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:29:51.823+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:29:51.847+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:29:51.842+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:29:51.850+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:29:51.872+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.058 seconds
[2024-10-11T14:30:22.303+0000] {processor.py:186} INFO - Started process (PID=1772) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:30:22.306+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:30:22.308+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:30:22.308+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:30:22.431+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:30:22.423+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:30:22.437+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:30:22.466+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.169 seconds
[2024-10-11T14:30:52.943+0000] {processor.py:186} INFO - Started process (PID=1835) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:30:52.945+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:30:52.947+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:30:52.947+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:30:52.967+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:30:52.963+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:30:52.970+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:30:52.990+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.053 seconds
[2024-10-11T14:31:23.165+0000] {processor.py:186} INFO - Started process (PID=1898) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:31:23.166+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:31:23.169+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:31:23.169+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:31:23.200+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:31:23.192+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:31:23.203+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:31:23.245+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.086 seconds
[2024-10-11T14:31:53.851+0000] {processor.py:186} INFO - Started process (PID=1961) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:31:53.852+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:31:53.855+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:31:53.854+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:31:53.883+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:31:53.877+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:31:53.885+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:31:53.909+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.064 seconds
[2024-10-11T14:32:23.969+0000] {processor.py:186} INFO - Started process (PID=2024) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:32:23.971+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:32:23.973+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:32:23.973+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:32:24.021+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:32:24.002+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:32:24.026+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:32:24.055+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.090 seconds
[2024-10-11T14:32:54.499+0000] {processor.py:186} INFO - Started process (PID=2087) to work on /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:32:54.500+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/stock_scraper_dag.py for tasks to queue
[2024-10-11T14:32:54.503+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:32:54.502+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:32:54.529+0000] {logging_mixin.py:190} INFO - [2024-10-11T14:32:54.524+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/stock_scraper_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/stock_scraper_dag.py", line 8, in <module>
    from scrapers.stock_data_scraper import StockDataScraper
ModuleNotFoundError: No module named 'scrapers'
[2024-10-11T14:32:54.531+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/stock_scraper_dag.py
[2024-10-11T14:32:54.556+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/stock_scraper_dag.py took 0.064 seconds
